client/src/plus/notifications/notifications-tab.tsx

```diff
@@ -164,6 +164,29 @@ export function NotificationsTab({
     const updated = list.filter((v) => !v.checked);
     setList(updated);
   };
+  let empty_text = !starred ? (
```

Comment:
Any reason to use `let` here?

```suggestion
  const empty_text = !starred ? (
```